packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -148,6 +152,7 @@ export default abstract class SqlIntegration
       pastExperiments: true,
       supportsInformationSchema: true,
       supportsAutoGeneratedMetrics: this.isAutoGeneratingMetricsSupported(),
+      supportsAutoGeneratedFactTables: this.isAutoGeneratingFactTablesSupported(),
```

Comment:
I could combine these two into a single `supportsAutoGeneratingResources` if we think these two will always be in sync.